Card/Event/Cancel.ts

```diff
@@ -0,0 +1,13 @@
+import { isoly } from "isoly"
+import { isly } from "isly"
+
+export interface Cancel {
+	type: "cancel"
+	created: isoly.DateTime
```

Comment:
This feels weird, should we go with timestamp instead of created for all of them?

Reply:
maybe "at"?

Reply:
Sounds good!